### DISCUSSION TITLE
How are PCI devices managed in Talos?

### DISCUSSION THREAD

**Discussion by roeeklinger** [siderolabs/talos#7219] [Q&A]
I am looking to move away from Proxmox to Talos Bare Metal, My server has 5 PCIe cards on it, each card has six 4G modules.

On Proxmox, I can easily go to the VM configuration page, and select which PCI card goes to which machine, however on Talos I cannot find any info on how this is managed.
Example: https://pve.proxmox.com/wiki/PCI(e)_Passthrough

If I install Talos bare metal on my server, how can I manage the PCIe cards, and decide which pod can see which card?

**Comment by frezbo** [comment]
You'd have to use a specific device plugin for the hardware https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/
